src/handlers/rendered_link.rs

```diff
@@ -34,65 +36,83 @@ async fn add_rendered_link(ctx: &Context, e: &IssuesEvent, prefix: &str) -> anyh
     if e.action == IssuesAction::Opened
         || e.action == IssuesAction::Closed
         || e.action == IssuesAction::Reopened
+        || e.action == IssuesAction::Synchronize
```

Comment:
I originally though I would have to go through the `PushEvent` instead of the `Issue/PushRequestEvent`, but it seems to be as simple as checking the `synchronize` action, yay for simplicity.